cmd/scw/main.go

```diff
@@ -25,11 +25,12 @@ var (
 
 func main() {
 	// Import all commands available in CLI from various packages.
+	// NB: Merge order impacts scw usage sort.
```

Comment:
We should add a test with golden for `scw -h`

Reply:
This would mean copy pasting this list of merge. I don't see a good way of doing it.

Reply:
is a function `getCommands()` overkill ? 
Should the `main`directly do the merging ?